modules/microshift-manifests-upgrade-scenario-ostree.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * microshift/running_applications/microshift-deleting-resource-manifests.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-manifests-upgrade-scenario-ostree_{context}"]
+= Upgrading manifests for OSTree systems
+
+Use the following procedure to remove some resources while retaining others to preserve data.
+
+[IMPORTANT]
+====
+For OSTree systems, the `delete` subdirectories are read only.
+====
+
+.Procedure
+
+. Identify the manifest that needs updating.
+. Create a new manifest for resource deletion to be placed in the `delete` subdirectories.
+. Create a new manifest to apply in the manifest path. See link:https://docs.redhat.com/en/documentation/red_hat_build_of_microshift/{ocp-version}/html/running_applications/applications-with-microshift#microshift-applying-manifests-example_applications-microshift[Using manifests example] to create new manifests using the example.
+. Use the procedure in "Removing manifests for OSTree systems" to create `delete` subdirectories and place the manifests created for resource deletion in this path.
```

Comment:
The _Removing manifests for OSTree systems_ procedure doesn't seem to show how to "create `delete` subdirectories...". This would confuse me as a user, where I wouldn't be sure I was in the right place.

Reply:
I have changed the sentence.